### PR TITLE
Add --no-merge flag

### PR DESCRIPTION
**Pull request by ModST**
added a --no-merge flag since with some formats we needed to merge the files ourselves.

### CODE REVIEW

**Comment by grosser** [comment]
❤️

File: spec/parallel_split_test_spec.rb [hunk 1]
```diff
@@ -270,7 +269,29 @@ def time
...
+        result = parallel_split_test "xxx_spec.rb --format d --out xxx --no-merge"
+        # output does not show up in stdout
+        expect(result).not_to include "xxx"
+        expect(result).not_to include "yyy"
+
+        # basic output is still there
+        expect(result).to include "Running examples in"
+
+        # two separate out files remain
+        expect(Dir["*-xxx"]).to eq(["0-xxx", "1-xxx"])
```
Comment (grosser) [inline spec/parallel_split_test_spec.rb]: order is different on travis -> sort both before comparing

also feel free to drop 1.8/1.9 support from travis.yml ... ideally add 2.2 / 2.3 support :)

**Comment by ModST** [comment]
Not sure why that build keeps failing with ruby 1.9

**Comment by bootstraponline** [comment]
> Not sure why that build keeps failing with ruby 1.9

I think we should drop 1.9.

**Comment by grosser** [comment]
thx, 0.5.0 is out

**Pull request merged**